osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs

```diff
@@ -108,33 +111,67 @@ public async Task ProcessScoreAsync(SoloScoreInfo score, MySqlConnection connect
             if (!score.Passed)
                 return;
 
-            beatmapStore ??= await BeatmapStore.CreateAsync(connection, transaction);
+            long currentTimestamp = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+
+            if (buildStore == null || beatmapStore == null || currentTimestamp - lastStoreRefresh > 60)
```

Comment:
Uh..... if this meant to refresh the store every minute, then that's not what this says is it? This refreshes the store every 60 _milliseconds_.

```suggestion
            if (buildStore == null || beatmapStore == null || currentTimestamp - lastStoreRefresh > 60_000)
```

is what you probably want if you intended the former.